### PR TITLE
Sort imports with isort

### PR DESCRIPTION
**Pull request by lioramilbaum**

### CODE REVIEW

**Comment by mprahl** [comment]
@lioramilbaum this is a cool tool. One of the files seems to have some style changes done to it as well. Could you please exclude those?

FYI, I'm also fine with using `black` to format everything.

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[settings]
+known_first_party = estuary
```
Comment (mprahl) [inline .isort.cfg:2]: Should this also include `scrapers` and `tests`?

Comment (mprahl) [reply .isort.cfg:2]: Why did you add estuary there then if it can auto detect?

Comment (lioramilbaum) [reply .isort.cfg:2]: because it didn't auto detect

**Pull request merged**